src/generate.js

```diff
@@ -291,7 +291,7 @@ export const generateCSSRuleset = (
     // Mutates declarations
     runStringHandlers(declarations, stringHandlers, selectorHandlers);
 
-    const originalElements = {...declarations.elements};
+    const originalElements = new Set(Object.keys(declarations.elements));
```

Comment:
Might be nice to add a check for this in dev since browser support is advertised at IE9 [/tools/generate_prefixer_data.js#L14](https://github.com/Khan/aphrodite/blob/master/tools/generate_prefixer_data.js#L14). This could be missed be some and not caught until consumers have reported it.

 Another possibility is to make this a major release since it is technically breaking for IE9 and 10.

Reply:
The next release will be a major version anyway, so we should be alright here. I like the idea of a check in dev but not really sure I see how that would be much more useful than it just erroring out normally. Thoughts?

Reply:
Major will be perfect.  Definitely looking forward to the next release. 

For the dev check, I was thinking more of a console warning.  Since I am sure most do not do local dev with <IE11, although I am not sure that it would add any benefit.  I was more hoping for the major version increase.